internal/ast/python/extractor.py

```
@@ -0,0 +1,487 @@
+                )
+
+        self.generic_visit(node)
+
+    def _extract_call_name(self, node: ast.Call) -> Optional[str]:
+        try:
+            if isinstance(node.func, ast.Name):
+                return node.func.id
+            elif isinstance(node.func, ast.Attribute):
+                return ast.unparse(node.func)
+            else:
+                return ast.unparse(node.func)
```

Pylint (3.3.7)

[refactor] Unnecessary "elif" after "return", remove the leading "el" from "elif"

(R1705)